src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java

```diff
@@ -407,14 +404,6 @@ public ExportNetworkInfos exportCgmesSv(UUID networkUuid) throws XMLStreamExcept
     private static CgmesExportContext createContext(Network network) {
         CgmesExportContext context = new CgmesExportContext();
         context.setScenarioTime(network.getCaseDate());
-        Optional<CgmesMetadataModel> cgmesMetadataModelOpt = network.getExtension(CgmesMetadataModels.class).getModelForSubset(CgmesSubset.STATE_VARIABLES);
-        if (cgmesMetadataModelOpt.isPresent()) {
-            context.getExportedSVModel().addDependentOn(cgmesMetadataModelOpt.get().getId());
-        }
-        cgmesMetadataModelOpt = network.getExtension(CgmesMetadataModels.class).getModelForSubset(CgmesSubset.STEADY_STATE_HYPOTHESIS);
-        if (cgmesMetadataModelOpt.isPresent()) {
-            context.getExportedSSHModel().addDependentOn(cgmesMetadataModelOpt.get().getId());
-        }
```

Comment:
I don't understand why we delete this part. There is no mention of this in the migration guide or i'm missing it ?